src/containers/Main/style.css

```diff
@@ -2,4 +2,10 @@
     width: 100%;
     height: 100%;
     overflow: auto;
+    background-image: url('../../assets/images/Book\ pages.jpg');
+    background-size:cover;
```

Comment:
non footer related changes would be better in another branch

Reply:
ALthough it is a non-footer change, it was necessary to get the footer to had padding-top - would you rather me move this change to another branch still?

Reply:
Hmm maybe I didn't switch branches? I'll have to check this later if you still want me to move it to a different branch and re-push